client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -31,6 +33,9 @@ export default function Dashboard(): JSX.Element {
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
       <CssBaseline />
       <Grid item className={classes.drawerWrapper}>
+        <Link to={'./settings/editProfile'}>
```

Comment:
Could this be just `/settings/editProfile`?

Reply:
OK